src/UI/Tooltip.js

```diff
@@ -4,7 +4,10 @@ const {getInputMainType} = require('../Form/input-classifiers')
 const updatePosition = function ({left, top}) {
     const shadow = this.shadow
     // If the stylesheet is not loaded wait for load (Chrome bug)
-    if (!shadow.styleSheets.length) return this.stylesheet.addEventListener('load', this.checkPosition)
+    if (!shadow.styleSheets.length) {
+        this.stylesheet.addEventListener('load', this.checkPosition)
+        return
+    }
```

Comment:
Just cosmetic, but makes it easier to read the keeps the return points consistent.